### PR TITLE
Bump chart-operator channel and chartconfig version bundle version

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

**Comment by fgimenez** [comment]
@rossf7 good point, done PTAL.

File: service/controller/aws/v6/version_bundle.go [hunk 1]
```diff
@@ -9,8 +9,13 @@ func VersionBundle() versionbundle.Bundle {
 		Changelogs: []versionbundle.Changelog{
 			{
 				Component:   "cluster-operator",
-				Description: "Add your changes here.",
-				Kind:        versionbundle.KindAdded,
+				Description: "Set chart-operator channel to 0-2-stable.",
```
[the remark this answers is not in the document]
Comment (fgimenez) [reply service/controller/aws/v6/version_bundle.go]: Much better :) Done and CI green on this build https://circleci.com/workflow-run/2b4ed5d6-8cd1-460d-8942-6bbc6872de31, merging.

File: service/controller/aws/v6/version_bundle.go [hunk 2]
```diff
@@ -9,7 +9,7 @@ func VersionBundle() versionbundle.Bundle {
 		Changelogs: []versionbundle.Changelog{
 			{
 				Component:   "cluster-operator",
-				Description: "Add your changes here.",
+				Description: "Added monitoring for charts managed by chart-operator.",
```
Comment (rossf7) [inline service/controller/aws/v6/version_bundle.go]: Sorry one more change. Can you make the component chart-operator.

Comment (fgimenez) [reply service/controller/aws/v6/version_bundle.go]: Good point, done.

**Pull request merged**